editor/code/package.json

```diff
@@ -88,6 +88,10 @@
       {
         "command": "coq-lsp.goals",
         "title": "Coq LSP: Show Goals at Point"
+      },
+      {
+        "command": "coq-lsp.document",
+        "title": "Coq LSP: Get document in serialized format, print in console"
```

Comment:
Do we want to expose this in the client?

Reply:
Not sure, maybe we could indeed safe the stuff in a file. For now I need it for testing easily.